Assets/HoloToolkit/Utilities/Scripts/Inspectors/MRTKEditor.cs

```diff
@@ -28,7 +28,17 @@ public class MRTKEditor : Editor
     {
         #region static vars
         // Toggles custom editors on / off
-        public static bool ShowCustomEditors { get; private set; }
+        public static bool ShowCustomEditors
+        {
+            get
+            {
+                return GetEditorPref("MRTK_Show_MRTK_Editors", true);
+            }
+            private set
+            {
+                SetEditorPref("MRTK_Show_MRTK_Editors", value);
```

Comment:
Probably best to make this a const string with an underscore prefix.

Reply:
Why put MRTK twice in the string?

Reply:
It's moot now - it was the equivalent of a project prefix, but your utility's approach is better.